spec/stack.js

```diff
@@ -11,12 +11,48 @@ describe('Stack', () => {
     expect(Stack).to.be.a('function')
   })
 
+  context('length()', () => {
+    it('returns the size of the stack', () => {
+      const myStack = new Stack()
+      myStack.push('foo')
+      myStack.push('bar')
+      expect(myStack.length()).to.eql(2)
+    })
+  })
+
+  context('isEmpty()', () => {
+    it('returns true if the stack is empty', () => {
```

Comment:
This test description does not match the expectation under test.  Consider adding both the `true if empty` and `false if not empty` cases.